galleries/examples/ticks/multilevel_ticks.py

```diff
@@ -0,0 +1,99 @@
+"""
+=========================
+Multilevel (nested) ticks
+=========================
+
+Sometimes we want another level of tick labels on an axis, perhaps to indicate
+a grouping of the ticks.
+
+Matplotlib does not provide an automated way to do this, but it is relatively
+straightforward to annotate below the main axis.
+
+These examples use `.Axes.secondary_xaxis`, which is one approach. It has the
+advantage that we can use Matplotlib Locators and Formatters on the axis that
+does the grouping if we want.
+
+This first example creates a secondary xaxis and manually adds the ticks and
+labels using `.Axes.set_xticks`.  Note that the tick labels have a newline
+(e.g. `"\nOughts"`) at the beginning of them to put the second-level tick
```

Comment:
```suggestion
(e.g. ``"\nOughts"``) at the beginning of them to put the second-level tick
```
Sphinx is trying to create a reference from this one.

Reply:
Feel free to self-merge once that's fixed.